src/lib/utils.js

```diff
@@ -37,3 +38,17 @@ exports.themeSwitch = function(object) {
 
   return retValue;
 };
+
+/**
+ * Confirm prompt service.
+ *
+ * @param {String} title - title of confirm prompt
+ * @param {String} message - question
+ * @returns {boolean} TRUE if user confirm, FALSE otherwise
+ */
+exports.confirm = function(title, message) {
+  let prompts = chrome.Cc["@mozilla.org/embedcomp/prompt-service;1"]
```

Comment:
`chrome.Cc` => `Cc` and `chrome.Ci` => `Ci` since we've initialized the constants above. I'm a bit worried that there is no SDK API for that, but oh well.